.circleci/config.yml

```diff
@@ -48,16 +48,23 @@ jobs:
           path: ./test-results
 
   e2e-test:
+    working_directory: ~/vets-website
     docker:
       - image: cypress/base:10.15.3
     parallelism: 2
-    environment:
     steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - node-v1-{{ .Branch }}-{{ checksum "package.json" }}
```

Comment:
Why isn't the Yarn cache restored here tooo?

Reply:
Restoring the Yarn cache here didn't change the behavior when I was testing. Since this step isn't saving any cache, I just left it out.

Reply:
Cypress and Yarn cache are now restored in `e2e-test`.